lib/em-hiredis/connection_manager.rb

```diff
@@ -57,6 +63,11 @@ def initialize(connection_factory, em = EM)
       @sm.on(:failed, &method(:on_failed))
     end
 
+    def _parse_options(options)
```

Comment:
Did you add a `_` to indicate it's a private method? If so, I think a better approach is to put this under a `private` section in this class.